ash/src/lib.rs

```diff
@@ -33,6 +33,12 @@
 //! Load the Vulkan library linked at compile time using [`Entry::linked()`], or load it at runtime
 //! using [`Entry::load()`], which uses `libloading`. If you want to perform entry point loading
 //! yourself, call [`Entry::from_static_fn()`].
+//!
+//! ## Create features
+//!
+//! * __debug__ (default): Whether Vulkan structs should implement Debug.
```

Comment:
These are underscores (`__`), not backticks. But as said, if other crates use `**` (same effect as `__`) we should probably too.

Reply:
ah my bad, should have read your comment properly...